src/pages/VisualizationPage.js

```diff
@@ -3,16 +3,22 @@ import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 import BalanceHistory from '../components/BalanceHistory';
+import PartyGrouping from '../components/PartyGrouping';
 
 const Container = styled.div`
-  margin: 3rem;
   text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 60rem;
+  margin: 3rem auto 3rem;
 `;
 
 const Visualization = ({ initialTransactions }) => (
   <Container>
```

Comment:
instead of hand writing flex related styles, I suggest looking in Grid component from material.